modules/zivid/handeye.py

```diff
@@ -0,0 +1,125 @@
+"""Contains hand-eye calibration functions and classes."""
+import _zivid
+
+
+def detect_feature_points(point_cloud):
+    """Detect feature points from a calibration object in a point cloud.
+
+    The functionality is to be exclusively used in combination with Zivid verified checkerboards.
+    For further information please visit Zivid help page.
+
+    Args:
+        point_cloud: cloud from a frame containing an image of a calibration object
+
+    Returns:
+        Instance of DetectionResult
+
+    """
+    return DetectionResult(
+        _zivid.handeye.detect_feature_points(
+            point_cloud._PointCloud__impl  # pylint: disable=protected-access
+        )
+    )
+
+
+def calibrate_eye_in_hand(calibration_inputs):
```

Comment:
This one doesn't actually work, because you have forgotten to add it to `HandEye.cpp`

Reply:
It works now, since the last fixups.